internal/node/node.bzl

```diff
@@ -119,11 +119,12 @@ def _write_loader_script(ctx):
         is_executable = True,
     )
 
-def _short_path_to_manifest_path(ctx, short_path):
-    if short_path.startswith("../"):
-        return short_path[3:]
+# Avoid writing non-normalized paths (workspace/../other_workspace/path)
+def _to_manifest_path(ctx, file):
```

Comment:
note to self: there are a few other places that we have _short_path_to_manifest_path and a few that are named _file_to_manifest_path